### DISCUSSION TITLE
GraphQL or REST fastAPI

### DISCUSSION THREAD

**Discussion by Matthieu-Tinycoaching** [fastapi/fastapi#4009] [Questions]
Hi community,

I have a quick question that turns me on for a while. I work with both graphQL and fastAPI, however I can see through documentation and tutorials that REST API are by far more usual. Are they definitely better than graphQL? Since I just started my graphQL API I wonder whether I should move on REST or keep going on with strawberry?
Thanks for your lights!

**Comment by baby5** [comment]
FastAPI is created to write api quickly for AI(tensorflow, pytorch...) model interface server.
For CURD app, graphQL and RESTful each has its own advantages and disadvantages.
You can get more info by google “graphQL vs REST”.
Which one to use depends on you.
